__tests__/app.test.js

```diff
@@ -137,6 +137,96 @@ describe("/api/articles/:article_id", () => {
             })
         })
     })
+    describe("PATCH", () => {
+        test("PATCH 200: Responds with an article object of the given ID", () => {
+            const newVote = {inc_votes: 1}
+            return request(app)
+            .patch("/api/articles/1")
+            .send(newVote)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
You know the article ID, actually, you know the entire article. Where we can we should be specific. We have little use for `any`s here

Reply:
Makes sense, cheers

Reply:
Cool, makes sense